src/lib/parts.ts

```diff
@@ -58,12 +85,33 @@ export class AttributeCommitter {
 
   protected _getValue(): any {
     const strings = this.strings;
+    const parts = this.parts;
     const l = strings.length - 1;
+
+    // If we're assigning an attribute via syntax like:
+    //    attr="${foo}"  or  attr=${foo}
+    // but not
+    //    attr="${foo} ${bar}" or attr="${foo} baz"
+    // then we don't want to coerce the attribute value into one long
+    // string. Instead we want to just return the value itself directly,
+    // so that sanitizeDOMValue can get the actual value rather than
+    // String(value)
+    if (l === 1 && strings[0] === '' && strings[1] === '' && parts[0]) {
+      const v = parts[0].value;
+      if (Array.isArray(v)) {
+        if (v.length === 1) {
+          return v[0];
+        }
```

Comment:
There's a fall-though here that I want to make sure is intentional. Can you comment it?